lib/vanagon/component.rb

```diff
@@ -140,10 +141,13 @@ def get_build_dir
     #
     # @param workdir [String] working directory to put the source into
     def get_sources(workdir)
-      @sources.each do |source|
-        cur_source = Vanagon::Component::Source.source(source.url, { :ref => source.ref, :sum => source.sum }, workdir)
-        cur_source.fetch
-        cur_source.verify
+      sources.each do |source|
```

Comment:
Am I missing something that ruby does to resolve the source variable here? From what it looks like the sources var is only in scope to the get_sources function and it's not defined yet in this function.